hera_mc/mc_session.py

```diff
@@ -2887,52 +2886,15 @@ def _get_node_snap_from_serial(self, snap_serial, session=None):
             return None, None
 
         if len(conn_dossier.keys()) > 1:
-            warnings.warn('Multiple active dossiers returned for snap serial '
-                          '{snn}. Setting node and snap location numbers to '
-                          'None'.format(snn=snap_serial))
+            warnings.warn('Multiple {} snaps were found, please specify the revision. '
+                          'Setting node and snap location numbers to None'
+                          .format(snap_serial))
```

Comment:
There's no way to specify the revision to this method. We could add one, but this method is not generally going to be called by humans, it's going to be called by automated code that is just trying to resolve a snap serial number from the correlator redis into a node and snap location number. So I think we want the defaulting to give you the right information and I don't really see how the code could/should specify a revision.

Also, I find the change from `active dossiers` to `snaps` to be confusing. Presumably a single snap serial number doesn't resolve to several snaps, but if the database has issues it can resolve to several connections/dossiers/whatever-you-call-them.

If this can never happen because of the improvements you've put in place that make the database more stable, lets take it out. On the other hand, if it can happen but the way it happens is misrepresented in the test I wrote, we should fix the test to make it match a more real world scenario.

Reply:
Added rev to call.  Defaults to None.

I'm not sure what is meant by the `active dossiers` to `snaps`